doc/getting-started.md

```diff
@@ -596,9 +649,46 @@ Alternatively you can use the powershell script on windows:
 ./upload-templates.ps1
 ```
 
-This creates the basic templates used by the OpenDevStack quickstarters in the `cd` project.
+This script creates the basic templates used by the OpenDevStack quickstarters in the `cd` project.
 If you have to modify templates, there are also scripts to replace existing templates in OpenShift.
 
+### Prepare CD project for Jenkins
+
+Now create secrets inside the CD project.
+
+```
+oc process -n cd templates/secrets -p PROJECT=cd | oc create -n cd -f-
+```
+
+We will now build base images for jenkins and jenkins slave:
+
+* Clone the [cicd project](https://github.com/opendevstack/cicd)
+* Customize the configuration in the `ods-configuration` project at **cicd > jenkins > ocp-config > bc.env**
+* Inside the cicd project execute `tailor`
+
+`tailor -n cd --template-dir jenkins/ocp-config --param-dir ../ods-configuration/cicd/jenkins/ocp-config update --selector app=jenkins`
```

Comment:
All of these commands are a simple `tailor update` now as the `Tailorfile` specifies all params.